Fix: README - Update (#544)
bluebanquise/bluebanquise: README.md

**BlueBanquise** is a coherent **Ansible** roles collection, designed to deploy and manage large group of hosts (clusters of nodes).

While main target is High Performance Computing (HPC), the BlueBanquise stack is generic and can adapt to any kind of architecture (university or enterprise infrastructures, render farm, etc.).

BlueBanquise is part of the **Algoric** Project from the [**Fabrique du Loch**](https://www.lafabriqueduloch.org/fr/accueil/) FabLab.

![BlueBanquise Logo](resources/pictures/FabriqueDuLochAlgoric_logo_large.svg)

It is a revamping of the old stack [Banquise](https://github.com/oxedions/banquise), based on Salt.

## Resources

### Files

The stack is split over multiple repositories:

> * :globe_with_meridians: **Core:** stack CORE roles and tools, dedicated to deploying and configuring servers, is hosted on github, in this [same repository](https://github.com/bluebanquise/bluebanquise).

Reviewer: ```suggestion
* :globe_with_meridians: **[Core](https://github.com/bluebanquise/bluebanquise):** the CORE of the stack, provides roles and tools to deploy and configure hosts.
```

Author: Ok for me.

> * :star: **Community:** stack COMMUNITY roles and tools, dedicated to bring more specialized features on top of CORE, is hosted on github, in the [community repository](https://github.com/bluebanquise/community).

Reviewer: ```suggestion
* :star: **[Community](https://github.com/bluebanquise/community):** COMMUNITY roles and tools, provides additional features on top of CORE. The release cycle of COMMUNITY is different than CORE.
```

Author: Ok for me.

### Documentation

The stack documentation is available on the BlueBanquise website, in [documentation subfolder](https://bluebanquise.com/documentation/).

Note that each role (CORE or COMMUNITY) embeds its own readme, with detailed
usage description.

### Packages

The stack packages are available in the [repositories subfolder](https://bluebanquise.com/repository/).

## Supported software environment

| Operating System family | Operating System distribution | Tested versions    | Notes                               |
| ----------------------- | ----------------------------- | ------------------ | ----------------------------------- |
| Red Hat                 |                               |                    |                                     |
|                         | RHEL                          | 7.x, 8.x           | Should work on all 7 and 8 versions |

> |                         | CentOS                        | 7.x, 8.x           | Should work on all 7 and 8 versions |

Author: Yup, I will update that.

|                         | CentOS Stream                 | NA                 | In dev, targeted for next release   |
|                         | Oracle Linux                  | NA                 | In dev, targeted for next release   |
|                         | CloudLinux                    | NA                 | In dev, targeted for next release   |
|                         | RockyLinux                    | NA                 | Waiting for distribution release    |

> |                         | Fedora                        | NA                 | In dev, targeted for next release   |

Author: It is already done in https://github.com/bluebanquise/bluebanquise/pull/542
I just need to adapt to current master.
